### ISSUE TITLE
q&v: fields missing for features view

### ISSUE THREAD

**Issue by luizaogs** [arch-kiosk/arch-kiosk-office#2357]
Well, in light of your email, I thought I would open this. 

Missing: feature type, measurements, formation process, materials, coordinates, revisit this feature one day because, recorded in, interpretation, tags.

**Comment by luizaogs** [comment]
This is after testing with a complete record so this is all that is missing.

**Comment by urapadmin** [comment]
features are postponed for now

**Comment by urapadmin** [comment]
- same problem as #2358

**Comment by urapadmin** [comment]
### Kiosk 1.7.27
- feature units are at least shown now and list their own fields: type of feature, material, revisit, dimensions and created/modified/modified by
- the view also shows the fields bound to the invisible locus without showing it as a locus. Yeah. That works.

**Comment by luizaogs** [comment]
<img width="207" height="61" alt="Image" src="https://github.com/user-attachments/assets/f13d95ae-066e-4025-9683-2919993853a6" />

Don't know if this matters, but these are two separate materials. In Q&V it looks like one thing because there is nothing separating them.

**Comment by luizaogs** [comment]
Still missing: coordinates, recorded in, tags.

I assume these are fields that you can't add to this version of Q&V. Let me know if you want this to be closed (if the above issue isn't an issue) and a new ticket opened, or just for this to be left open.

**Comment by urapadmin** [comment]
> 
> Don't know if this matters, but these are two separate materials. In Q&V it looks like one thing because there is nothing separating them.

Ja, all I could do is throw this out entirely. It is one of the multiline filemaker values that need special treatment that Q&V cannot provide in the current version.

**Comment by urapadmin** [comment]
actually, there was a somewhat unusual way to pull this off.
### Kiosk 1.7.28.1
- material in features should be multi line now
- coordinates should be there now
- recorded_in cannot be added at this point
- no tags either

**Comment by luizaogs** [comment]
Ticket for next iteration of Q&V: #3476

**Comment by luizaogs** [comment]
Will test coordinates when I can synchronize.

**Comment by luizaogs** [comment]
Looks good, too.
